### PR TITLE
(BOLT-252) change raketask to work with LAYOUT ENV

### PR DESCRIPTION
**Pull request by zreichert**

### CODE REVIEW

File: acceptance/Rakefile [hunk 1]
```diff
@@ -2,29 +2,19 @@ require 'rototiller'
...
-    roles << 'bolt' if i.zero?
-    roles << if /win/ =~ node
-               'winrm'
-             else
-               'ssh'
-             end
-    node = node + roles.join(',') + '.'
-    n_new << node
+
+  task.add_env(name: 'LAYOUT', message: <<-MESSAGE)
```
Comment (adreyer) [inline acceptance/Rakefile]: can we get a default or some examples?

**Comment by adreyer** [comment]
You can add a test that makes sure the task_wrapper isn't called if it's not going to sudo

File: acceptance/Rakefile (hunk 1 above)
Comment (zreichert) [reply acceptance/Rakefile]: Sure

**Comment by MikaelSmith** [comment]
It looks like the README should've been updated.

**Pull request merged**